stdlib/task/src/main/ballerina/src/task/scheduler.bal

```diff
@@ -65,7 +65,7 @@ public type Scheduler object {
 
     # Stops the task. This will stop, after finish running the existing jobs.
     #
-    # + return - Returns `task:SchedulerError` if the process failed due to any reason, nil otherwise.
+    # + return - `task:SchedulerError` if the process failed due to any reason, nil otherwise
```

Comment:
```suggestion
    # + return - `task:SchedulerError` if the process failed due to any reason, nil otherwise
```
Same as above. Also, please refer the template for the correct format of the return statement description.